src/main/java/nextstep/sessions/domain/Students.java

```diff
@@ -0,0 +1,41 @@
+package nextstep.sessions.domain;
+
+import java.time.LocalDateTime;
+import java.util.Set;
+import nextstep.users.domain.NsUser;
+
+public class Students {
+
+  private Set<Student> students;
+
+
+  public Students(Set<Student> students) {
+    this.students = students;
+  }
+
+  public boolean contains(Session session, NsUser user) {
+    return students.stream()
+        .filter(student -> student.isTaking(session, user))
+        .findAny()
+        .isPresent();
+  }
```

Comment:
```suggestion
  public boolean contains(Student student) {
    return students.contain(student);
  }
```
Session에서 Student를 생성해 전달하면 위와 같이 간단하게 구현이 끝나지 않을까?